source/clear-linux/get-started/virtual-machine-install/virtual-machine-install.rst

```diff
@@ -0,0 +1,36 @@
+.. _virtual-machine-install:
+
+Install Clear Linux under a Virtual Machine
```

Comment:
Similarly, I would change "under a Virtual Machine" to "in a virtual machine". Whichever the case, virtual machine must be in lower case.